app/controllers/sites_controller.rb

```diff
@@ -18,7 +18,13 @@ def show
   end
 
   def update
-    if @site.update(site_params)
+    @site.attributes = site_params
+    if @site.terms_of_use_content_changed?
+      User.update_all(terms_of_use_accepted: false)
+      @site.terms_of_use_changed_date = DateTime.now
+    end
+
+    if @site.save
```

Comment:
I see one little issue here - What if theoretically speaking saving failed. In this case, we would still set `terms_of_use_accepted` to `false` for all users. 
This is not a big issue but it could be maybe addressed as well.

Reply:
There's a convention to end date fields with `_at`. Maybe you rename this to `site.terms_of_use_changed_at` to make the code more readable.

Reply:
Ok. when i changed logic to dates it is unnecessarily now.

i changed column names to _at